front/lib/resources/storage/models/labs_transcripts.ts

```diff
@@ -69,6 +71,10 @@ LabsTranscriptsConfigurationModel.init(
       allowNull: false,
       defaultValue: false,
     },
+    dataSourceId: {
```

Comment:
Don't you want a foreign key here?

Reply:
We def want a foreign key and a number here :+1:

Reply:
Done